Cargo.toml

```diff
@@ -16,3 +23,59 @@ members = [
 	"ethereum-types",
 	"ethbloom",
 ]
+
+[workspace.dependencies]
+serde_json = "1.0.41"
+criterion = "0.5.1"
+rand = { version = "0.8.0", default-features = false }
+hex-literal = "0.4.1"
+scale-info = { version = ">=1.0, <3", default-features = false }
```

Comment:
this was `>= 0.9, < 3`, why did you change it? the reason we set it this way is to allow upsteam users to update `scale-info` without having to update crates in `parity-common`.

Reply:
ah, i see. ok, let's keep it `>= 1.0, < 3`. i think nobody should be using scale-info 0.9 at this point